doc/requirements.txt

```diff
@@ -1 +1,9 @@
 ptyprocess
+sphinxcontrib_github_alt
+
+# For sphinxcontrib_github_alt==1.1 due to missing install_requires
+# https://github.com/jupyter/sphinxcontrib_github_alt/issues/3
+# BEGIN
+docutils
+sphinx
+# END
```

Comment:
Let's just add Sphinx to the list, without the comment - Sphinx is the documentation tool, so it's needed regardless of what extensions we use, and it depends on docutils.

Reply:
Alright, done.